addon/ng2/blueprints/enum/files/__path__/__name__.spec.ts

```diff
@@ -0,0 +1,14 @@
+import {
+  describe,
+  ddescribe,
+  expect,
+  iit,
+  it
+} from '@angular/core/testing';
+import {<%= classifiedModuleName %>} from './<%= fileName %>';
+
+describe('<%= classifiedModuleName %>', () => {
+  it('should properly exist', () => {
+    expect(<%= classifiedModuleName %>).toBeTruthy();
+  });
+});
```

Comment:
It is not necessary to add a spec file for enums, nothing is executed so there is no logic to test.